net/portmapper/portmapper.go

```diff
@@ -55,6 +55,8 @@ type Client struct {
 	logf         logger.Logf
 	ipAndGateway func() (gw, ip netaddr.IP, ok bool)
 	onChange     func() // or nil
+	testPxPPort  uint16 // if non-zero, pxpPort to use
+	testUPnPPort uint16 // if non-zero, uPnPPort to use
```

Comment:
can we specify in comments these are only set in tests?

Reply:
Done. Although I thought the field name made that clear. Now it's double clear :)